# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../blender/openal.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- blender/SConstruct.vanilla	2006-03-05 10:48:24.821309250 +0100
+++ blender/SConstruct	2006-03-05 10:50:01.411345750 +0100
@@ -146,7 +146,7 @@
 	gettext_libpath = []
 	gettext_include = []
 	# OpenAL library information
-	openal_lib = ['openal']
+	openal_lib = ['openal', 'alut']
 	openal_libpath = ['/usr/lib']
 	openal_include = ['/usr/include']
 
